cmd/coordinator.go

```diff
@@ -264,6 +270,101 @@ var estimateDepositsSweepFeeCommandDescription = "Estimates the satoshi " +
 	"legacy P2SH deposits. If the estimated fee exceeds the maximum fee " +
 	"allowed by the Bridge contract, the maximum fee is returned as result"
 
+var submitDepositSweepProofCommand = cobra.Command{
+	Use:              "submit-deposit-sweep-proof",
+	Short:            "submit deposit sweep proof",
+	Long:             submitDepositSweepProofCommandDescription,
```

Comment:
Why not to inline it, it is really short:

```suggestion
	Long:             "Submits deposit sweep proof to the Bridge contract",
```

Reply:
Done in 4424a48b58bb8c05fc323dc52302e9c44c796cc5.